s3pypi/__main__.py

```diff
@@ -3,77 +3,50 @@
 import argparse
 import logging
 import sys
+from pathlib import Path
 
-from s3pypi import __prog__, __version__
-from s3pypi.exceptions import S3PyPiError
-from s3pypi.package import Package
-from s3pypi.storage import S3Storage
+from s3pypi import __prog__, __version__, core
 
+logging.basicConfig()
 log = logging.getLogger()
 
 
-def create_and_upload_package(args):
-    package = Package.create(args.wheel, args.sdist, args.dist_path)
-
-    storage = S3Storage(
-        args.bucket, args.secret, args.region, args.bare, args.acl, args.profile
-    )
-
-    index = storage.get_index(package)
-    index.add_package(package, args.force)
-
-    storage.put_package(package, args.dist_path)
-    storage.put_index(package, index)
-
-
 def parse_args(args):
```

Comment:
Same as previous comment, returning the `ArgumentParser` from a function is nice for external tooling.